test/e2e/service.go

```diff
@@ -490,8 +493,14 @@ var _ = Describe("Services", func() {
 		if providerIs("gce", "gke") {
 			By("creating a static load balancer IP")
 			rand.Seed(time.Now().UTC().UnixNano())
-			staticIPName = fmt.Sprintf("e2e-external-lb-test-%d", rand.Intn(65535))
-			requestedIP, err = createGCEStaticIP(staticIPName)
+			// might need to try more than once for an unused staticIPName
+			for attempts := 0; attempts < maxAttempts; attempts++ {
+				staticIPName = fmt.Sprintf("e2e-external-lb-test-%d", rand.Intn(65535))
```

Comment:
ugh, still getting used to variable scopes in Go.  recovering C programmer here. new version is pushed.